### ISSUE TITLE
java.lang.IllegalStateException: Rule id is mandatory on ad hoc rule

### ISSUE THREAD

**Issue by nij4t-spro** [gretard/sonar-sql-plugin#22]
Error stacktrace:
```
WARN: Unexpected error adding issue on file sample.sql
java.lang.IllegalStateException: Rule id is mandatory on ad hoc rule
	at org.sonar.api.utils.Preconditions.checkState(Preconditions.java:49)
	at org.sonar.api.batch.sensor.rule.internal.DefaultAdHocRule.doSave(DefaultAdHocRule.java:85)
	at org.sonar.api.batch.sensor.internal.DefaultStorable.save(DefaultStorable.java:45)
	at org.sonar.plugins.sql.sensors.BaseSensor.addIssues(BaseSensor.java:120)
	at org.sonar.plugins.sql.sensors.SqlCheckSensor.execute(SqlCheckSensor.java:100)
	at org.sonar.plugins.sql.sensors.SqlCheckSensor.access$000(SqlCheckSensor.java:23)
	at org.sonar.plugins.sql.sensors.SqlCheckSensor$1.run(SqlCheckSensor.java:62)
	at java.base/java.util.concurrent.ForkJoinTask$AdaptedRunnableAction.exec(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinPool.scan(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(Unknown Source)
```

Context:
```
Server Version:  8.5.1.38104

SonarQube plugins:
...
  - SQL language plugin 1.1.1 (sql)
...

Project scanner properties:
  - sonar.branch.name=master
  - sonar.host.url=***
  - sonar.projectBaseDir=/builds/example
  - sonar.projectKey=sql-dialects.pssql
  - sonar.qualitygate.timeout=300
  - sonar.qualitygate.wait=true
  - sonar.scanner.app=ScannerCLI
  - sonar.scanner.appVersion=4.6.2.2472
  - sonar.sourceEncoding=UTF-8
  - sonar.sources=sql
  - sonar.sql.dialect=pssql
  - sonar.sql.sqlcheck.path=/usr/bin/sqlcheck
  - sonar.working.directory=/builds/example/.scannerwork
  ```

**Comment by gretard** [comment]
Hi,

Please use the latest SonarQube version
